config/settings.yml

```diff
@@ -21,3 +21,4 @@ redis_cache_ssl: false
 slack_webhook: 'this is set through a env var in Openshift and not shared for security reasons'
 disable_rbac: 'false'
 async: true
+feature_comp_e_133: false
```

Comment:
I opted this time for a top level feature flag vs.

```yaml
features:
  comp_e_133: false
```

so it's easier to toggle with an env variable, SETTINGS__FEATURE_COMP_E_133